backend/src/SessionRepository.js

```diff
@@ -0,0 +1,26 @@
+class SessionRepository {
+	constructor( driver ) {
+		this._driver = driver;
+	}
+
+	async getAll( issueKey ) {
+		const issueSessions = {};
+		const response = await this._driver.client.hgetall( issueKey );
+
+		for ( const session in response ) {
+			issueSessions[ session ] = JSON.parse( response[ session ] );
+		}
+
+		return issueSessions;
+	}
+
+	async set( issueKey, socket, issueSession ) {
+		await this._driver.client.hset( issueKey, socket.id, JSON.stringify( issueSession ) );
+	}
+
+	async deleteOne( socket ) {
```

Comment:
Please try to not provide an object to the method. The method should expect specified parameters.
```
async deleteOne( issueKey, socketId ) {
```

Reply:
Also, check `set` method -> socket => socketId

Of course `issueSession` there is ok

Reply:
Fixed.